templates/_layouts/_footer.html

```diff
@@ -1,4 +1,4 @@
-<footer class="p-strip is-dark">
+<footer class="p-strip is-dark p-footer l-footer--sticky">
```

Comment:
Do we need both `p-footer` and `l-footer` here?

Reply:
I guess not, I will correct it.